### PR TITLE
29612 - page title & metadata update

### PR DESCRIPTION
**Pull request by zachary-horvath**

### CODE REVIEW

File: contentstack-finserv/app/experts/[slug]/page.tsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ export async function generateMetadata(
   return {
     title: `${
       data?.seo?.prefix
-        ? `${data?.config?.company_title} ${data?.seo?.meta_separator} ${data?.seo?.meta_title}`
-        : `${data?.seo?.meta_title} ${data?.seo?.meta_separator} ${data?.config?.company_title}`
+        ? `${data?.seo?.meta_company} ${data?.seo?.meta_title}`
```
Comment (mhak) [inline contentstack-finserv/app/experts/[slug]/page.tsx:25]: This should be called "Page Title Tag" field - conceivably they can add any text here, not just the company

**Review by mhak — CHANGES_REQUESTED** [review]

File: contentstack-finserv/app/experts/[slug]/page.tsx (hunk 1 above)
Comment (zachary-horvath) [reply contentstack-finserv/app/experts/[slug]/page.tsx:25]: Do you think this is confusing with the meta title field? Typically, meta titles do contain the company/app name in some capacity.

Comment (mhak) [reply contentstack-finserv/app/experts/[slug]/page.tsx:25]: feel free to come up with a better name, but it shouldn't just be "company", it should be any tag they want to add to the beginning or end of the page title

**Review by mhak — CHANGES_REQUESTED** [review]

File: contentstack-finserv/app/experts/[slug]/page.tsx (hunk 1 above)
Comment (mhak) [reply contentstack-finserv/app/experts/[slug]/page.tsx:25]: right - it _usually_ shows the company name, but let's not limit it to that. Let's call it "Page Tag" to match the a/c and kontent.ai

**Review by mhak — APPROVED** [review]

**Pull request merged**